neurodsp/tests/test_spectral_power.py

```diff
@@ -50,5 +74,20 @@ def test_compute_spectrum_wavelet(tsig):
 
 def test_compute_spectrum_medfilt(tsig):
 
+    # Create a sinusoid with 10 periods/cycles.
+    n_seconds = 10
+    fs = 100
+    sig = sim_oscillation(n_seconds, fs, 1, n_cycles=n_seconds, mean=None, variance=None)
```

Comment:
Same note on `n_cycles`, but also, since we re-use this signal, perhaps we should just add it as a fixture in conftest? It could be `tsig_sine_long` or something,  and use a `N_SECONDS_LONG` settings.